README.md

```diff
@@ -249,6 +249,7 @@ To ensure your environment is compatible, we also recommend the following VSCode
 
 **Number of files in a bulk upload:** Theoretically unlimited<br>
 **Max individual file size**: 2GB (Node.js limitation)<br>
+**Max file name length**: 254 characters<br>
```

Comment:
Isn't the limit 255 characters? That's what we agreed on in the ArFS Standards writeup...